src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -146,7 +146,7 @@ private void LoadParameters(ChainSpecJson chainSpecJson, ChainSpec chainSpec)
                 Eip3541Transition = chainSpecJson.Params.Eip3541Transition,
                 Eip3529Transition = chainSpecJson.Params.Eip3529Transition,
                 Eip3607Transition = chainSpecJson.Params.Eip3607Transition,
-                Eip1153Transition = chainSpecJson.Params.Eip1153Transition,
+                Eip1153TransitionTimestamp = chainSpecJson.Params.Eip1153TransitionTimestamp,
```

Comment:
Other shanghai EIPs?

Reply:
Will be added with their PRs or by merging shandong